src/app/modules/group/components/pending-request/pending-request.component.html

```diff
@@ -98,11 +97,11 @@
             <div class="req-tools">
               <span class="accept" [ngClass]="{disabled: selection.length === 0}" (click)="onAccept()">
                 <span>Accept</span>
-                <i class="fa fa-check" *ngIf="state !== 'accepting'; else loading"></i>
+                <i class="fa fa-check" *ngIf="state !== 'processing'; else loading"></i>
```

Comment:
No need for this spinner while processing, but what prevent the "accept" to be clicked while processing?

Reply:
fixed